src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Figure[] figure = new Figure[5];
```

Comment:
Instead of this array, use switch/case to get a random figure based on a random number